inject-groovy/src/test/groovy/io/micronaut/inject/configproperties/InheritedConfigurationReaderPrefixSpec.groovy

```diff
@@ -37,7 +37,7 @@ class MyBean  {
         beanDefinition.getInjectedMethods()[0].name == 'setMyValue'
         def metadata = beanDefinition.getInjectedMethods()[0].getAnnotationMetadata()
         metadata.hasAnnotation(Property)
-        metadata.getValue(Property, "name", String).get() == 'simple.my-value'
+        metadata.getValue(Property, "name", String).get() == 'endpoints.my-value'
```

Comment:
These tests are "testing" broken functionality of configuration path being remapped multiple times by aliases, these tests keep changing the output based on the annotation processing order.